practice/tic-tac-toe/coords_board.py

```diff
@@ -0,0 +1,166 @@
+"""Class Module for Tic-Tac-Toe game that handles the board."""
+
+
+class CoordsTTTBoard:
```

Comment:
Working through this, it seems this requires a lot of mapping within mapping?  Maybe I'm coming at this from the wrong angle, but I think the solution to simplifying this problem mirrors how I've simplified the Dictionary Board, but requires a little layered mapping in the creation of blank entries.

Specifically something like;

`[if (x, y) in item for item in self._token_coords:
    item,
else: 
    (x, y, ' ')`

but then iterating through a range for x and y?